src/registrar/forms/application_wizard.py

```diff
@@ -2,17 +2,16 @@
 from itertools import zip_longest
 import logging
 from typing import Callable
+from api.views import DOMAIN_API_MESSAGES
```

Comment:
We should add to the refactor: The error strings definitions whoul probably be at a lower level than the API. Maybe in utilities or in domain_helper